pkg/commands/standalone_server_test.go

```diff
@@ -129,9 +125,15 @@ func dnsLookup(t *testing.T, resolverPort, resolverProtocol, domain string) ([]n
 	return resolver.LookupIP(ctx, "ip4", domain)
 }
 
-//nolint:gosec
-func randomPort() string {
-	return fmt.Sprint(rand.Intn(32767) + 32768)
+// nolint: gocritic
```

Comment:
Please ensure the comment describes _which_ lint is being suppressed. That is, `gocritic` contains many lint rules; it's unclear which one it's violating.

If this is [`unamedresult`](https://go-critic.com/overview#unnamedresult), perhaps we should turn it off — it's off by default, and it doesn't sound very useful. I don't know why we turned it on in the first place.

Reply:
The `gocritic` is s useful one. I don't think we want that off. I left a comment for the reason which should be clear enough.